src/ucp/api/ucp.h

```diff
@@ -815,6 +816,19 @@ enum ucp_am_handler_param_field {
     UCP_AM_HANDLER_PARAM_FIELD_ARG     = UCS_BIT(3)
 };
 
+/**
+ * @ingroup UCP_COMM
+ * @brief UCP request priority level
+ * 
+ * The enumeration lists available priority levels which are set per request, 
+ * Message priority is an optional attribute of @ref ucp_request_param_t, 
+ * value will be propogated to transports which are capable of prioritizing messages.
+*/
+typedef enum {
+    UCP_PRIORITY_DEFAULT = UCS_BIT(0),
+    UCP_PRIORITY_HIGH    = UCS_BIT(1)
```

Comment:
why is it a flag? I think we can't combine different prio levels for one request

Reply:
Agree, my bad